src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
-}
+  let oddReturn = 0;
+
+  function sum(num1) {
+    oddReturn++;
+
+    return (num2) => {
+      if (oddReturn > 3 && oddReturn % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return num1 + num2;
+      }
```

Comment:
You don't need this `else`
```suggestion
        return num1 + num2;
```